flash/text/seq2seq/core/data.py

```diff
@@ -23,16 +23,41 @@
 from flash.core.data.io.input import Input, InputFormat
 from flash.core.data.io.input_transform import InputTransform
 from flash.core.data.io.output_transform import OutputTransform
+from flash.core.data.process import Deserializer
 from flash.core.data.properties import ProcessState
 from flash.core.utilities.imports import _TEXT_AVAILABLE, requires
-from flash.text.classification.data import TextDeserializer
 
 if _TEXT_AVAILABLE:
     import datasets
     from datasets import DatasetDict, load_dataset
     from transformers import AutoTokenizer, default_data_collator
 
 
+class Seq2SeqDeserializer(Deserializer):
+    @requires("text")
+    def __init__(self, backbone: str, max_length: int, use_fast: bool = True, **kwargs):
+        super().__init__()
+        self.backbone = backbone
+        self.tokenizer = AutoTokenizer.from_pretrained(backbone, use_fast=use_fast, **kwargs)
```

Comment:
Do you want to use the property too ?

Reply:
Will do when we get to Seq2Seq tasks in a follow up :smiley: